public/app/features/search/page/components/SearchResultsTable.test.tsx

```diff
@@ -0,0 +1,144 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { Subject } from 'rxjs';
+
+import { ArrayVector, DataFrame, DataFrameView, FieldType } from '@grafana/data';
+
+import { DashboardQueryResult, getGrafanaSearcher, QueryResponse } from '../../service';
+import { DashboardSearchItemType } from '../../types';
+
+import { SearchResultsTable } from './SearchResultsTable';
+
+describe('SearchResultsTable', () => {
+  const mockOnTagSelected = jest.fn();
+  const mockClearSelection = jest.fn();
+  const mockSelectionToggle = jest.fn();
+  const mockSelection = jest.fn();
+  const mockKeyboardEvents = new Subject<React.KeyboardEvent>();
+
+  describe('when there is data', () => {
+    const searchData: DataFrame = {
+      fields: [
+        { name: 'kind', type: FieldType.string, config: {}, values: new ArrayVector([DashboardSearchItemType.DashDB]) },
+        { name: 'uid', type: FieldType.string, config: {}, values: new ArrayVector(['my-dashboard-1']) },
+        { name: 'name', type: FieldType.string, config: {}, values: new ArrayVector(['My dashboard 1']) },
+        { name: 'panel_type', type: FieldType.string, config: {}, values: new ArrayVector(['']) },
+        { name: 'url', type: FieldType.string, config: {}, values: new ArrayVector(['/my-dashboard-1']) },
+        { name: 'tags', type: FieldType.other, config: {}, values: new ArrayVector([['foo', 'bar']]) },
+        { name: 'ds_uid', type: FieldType.other, config: {}, values: new ArrayVector(['']) },
+        { name: 'location', type: FieldType.string, config: {}, values: new ArrayVector(['/my-dashboard-1']) },
+      ],
+      length: 1,
+    };
+
+    const mockSearchResult: QueryResponse = {
+      isItemLoaded: jest.fn(),
+      loadMoreItems: jest.fn(),
```

Comment:
I tried:
```suggestion
      isItemLoaded: () => true,
      loadMoreItems: () => Promise.resolve(),
```
hoping that would help... but no luck.  I copied the rest of your tests/fixes and they work great in  https://github.com/grafana/grafana/pull/51208 :)